.gitignore

```diff
@@ -27,5 +27,4 @@
 *.mp4
 *.cpuprofile
 node_modules/
-build/webgazer.js
```

Comment:
Why was this removed? Shouldn't this be built by the user themselves using grunt.